.github/workflows/program-token-metadata.yml

```diff
@@ -29,6 +29,8 @@ jobs:
             - 'core/**'
           package:
             - 'token-metadata/**'
+  token-metadata-security-scan:
```

Comment:
What is this doing? I didn't see `token-metadata-security-scan` declared anywhere (I probably missed it or it already exists).